sdk/src/azure/core/az_mqtt5_rpc_client_codec.c

```diff
@@ -13,21 +13,23 @@
 #include <azure/core/_az_cfg.h>
 
 static const az_span az_mqtt5_rpc_client_codec_default_publish_topic_format
-    = AZ_SPAN_LITERAL_FROM_STR("services/{serviceId}/{executorId}/command/{name}/request");
+    = AZ_SPAN_LITERAL_FROM_STR(AZ_MQTT5_RPC_SERVER_DEFAULT_SUBSCRIPTION_TOPIC_FORMAT);
```

Comment:
If some of these are only used once, why not define the az_span there (ex. line 34 for this one)